README.md

```diff
@@ -155,9 +172,25 @@ PS> scoop bucket add extras
 PS> scoop install go-ipfs
 ```
 
+### Install prebuilt packages
+
+[![dist.ipfs.io Downloads](https://img.shields.io/github/v/release/ipfs/go-ipfs?label=dist.ipfs.io&logo=ipfs&style=for-the-badge&cacheSeconds=3600)](https://ipfs.io/ipns/dist.ipfs.io#go-ipfs)
+
+From there:
+- Click the blue "Download go-ipfs" on the right side of the page.
+- Open/extract the archive.
+- Move `ipfs` to your path (`install.sh` can do it for you).
+
+You can also download go-ipfs from this project's GitHub releases page if you are unable to access dist.ipfs.io:
+
+[![dist.ipfs.io Downloads](https://img.shields.io/github/v/release/ipfs/go-ipfs?label=GitHub%20Downloads&logo=ipfs&style=for-the-badge&cacheSeconds=3600)](https://github.com/ipfs/go-ipfs/releases)
```

Comment:
This is a bit of a gratuitous use of shields. I'd just turn "GitHub releases" into a link.

Reply:
Oh okay, I thought including the latest version would look fancy :)

I'll remove it.